src/components/Journal/JournalEntryForm/TagDimensionsGroup.tsx

```diff
@@ -0,0 +1,81 @@
+import { TagDimensionCombobox } from '../../../features/tags/components/TagDimensionCombobox'
+import { Tag } from '../../../features/tags/tagSchemas'
+import { useAvailableTagDimensions } from '../../../features/tags/api/useAvailableTagDimensions'
+import { FallbackWithSkeletonLoader } from '../../../components/SkeletonLoader/SkeletonLoader'
+
+const JOURNAL_ENTRY_FORM_CSS_PREFIX = 'Layer__JournalEntryForm'
+
+export interface TagDimensionsGroupProps {
+  dimensionKeys?: string[]
+  value: readonly Tag[]
+  onChange: (tags: readonly Tag[]) => void
+  showLabels?: boolean
+  isReadOnly?: boolean
+}
+
+export const TagDimensionsGroup = ({
```

Comment:
This is intended to be one compound component which handles 0+ "platform managed" dimensions. You render this element and it'll take care of rendering multiple inputs.